### ISSUE TITLE
Additional 'coloring' needed for ADA/WCAG accessibility.

### ISSUE THREAD

**Issue by lat9** [lat9/ZCA-Bootstrap-Template#210]
Additionally, the overall Bootstrap Colors' configuration needs a re-vamp so that it's not so painful next time when additional coloring for various elements is required/requested.

Things to consider:

1. If a site is upgrading as opposed to doing a fresh install, any new color settings need to be inserted.
2. Ditto with any various changes done to fix-up the sort-orders.
3. On the storefront side, if the updated `stylesheet_zca_colors.php` gets dropped into the storefront before an admin's had a chance to log in and get those defaults recorded, there are going to be (at best) PHP Warnings and (at worst) PHP Fatal errors due to those missing constants.

**Comment by lat9** [comment]
@dbltoe, I'm finding very _little rhyme or reason_ for the reason that some of the colors are bold and some aren't.   It seems (to me, anyway) that using a bold 'Title' for the first color in a grouping would be a good 'visual' hint.

![image](https://user-images.githubusercontent.com/2685585/219462707-9d0929fe-2ffa-481b-8dd3-905c03effc4e.png)

**Comment by dbltoe** [comment]
Yeah, it appears some were dragged over in a copy.

**Comment by lat9** [comment]
Ordering (if present) for each "group" will be updated (for consistency) and the first element in each "group" will be bold:

1. Background color
2. Background color on hover
3. Text color
4. Text color on hover
5. Link background color
6. Link background color on hover
7. Link color
8. Link color on hover
9. Button color
10. Button color on hover
11. Border color
12. Border color on hover

**Comment by dbltoe** [comment]
👍

**Comment by lat9** [comment]
Starting with v3.5.2, additional coloring settings will be 'versioned' and a "ZCA Bootstrap Colors" version will be added to the database.  Given that many sites currently use the Bootstrap template and have invested time in choosing their coloring scheme, simply adding and auto-enabling new colors will cause those sites to "look wonky".

As such, on an upgrade, any colors _added_ since the last install/upgrade of the "ZCA Bootstrap Colors" (as determined by the overall "ZCA Bootstrap Colors" version, will initially have their color-code recorded as `not-set`.  This will enable the storefront's `/css/stylesheet_zca_colors.php` to 'ignore' the application of those color settings until such time that the store has determined how best to incorporate those changes into their color-scheme.

The ignoring of any `not-set` value will apply *only* to colors added in v3.5.2 and later of the Bootstrap template.  Noting that the display in the admin of any color that is currently `not-set` will be a tad off, since that value's not a valid `#xxxxxx` hexadecimal value, with a color's configuration showing similar to:
![image](https://user-images.githubusercontent.com/2685585/219869313-650c21f0-d457-440d-8822-f69f5aa8865f.png)

The admin tool's table-display of all colors' will show the following when a color is 'not-set':
![image](https://user-images.githubusercontent.com/2685585/219869844-d49e404f-5b21-41be-998c-86d007a9f2a3.png)

**Comment by dbltoe** [comment]
👍
